storage/readservice/store.go

```
@@ -153,3 +153,12 @@ func (s *store) GetSource(orgID, bucketID uint64) proto.Message {
 		OrganizationID: orgID,
 	}
 }
+
+func (s *store) HasWindowAggregateCapability(ctx context.Context, capability ...*reads.WindowAggregateCapability) bool {
+	return false
+}
+
+// WindowAggregate will invoke a ReadWindowAggregateRequest against the Store.
+func (s *store) WindowAggregate(ctx context.Context, req *datatypes.ReadWindowAggregateRequest) (reads.ResultSet, error) {
+	return nil, nil
```

This should return an error.